repo/packages/Z/zeppelin/3/package.json

```diff
@@ -14,6 +14,6 @@
         "notebook",
         "interactive"
     ],
-    "website": "https://docs.mesosphere.com/zeppelin/",
+    "website": "https://github.com/mesosphere/dcos-zeppelin",
     "postInstallNotes": "DC/OS Zeppelin is being installed!\n\n\tDocumentation: https://docs.mesosphere.com/zeppelin/\n\tIssues: https://docs.mesosphere.com/support/"
```

Comment:
@eastlondoner Could you take care of the above change? I would be happy to merge it then.